client/galaxy/style/less/reports.less

```diff
@@ -207,3 +213,8 @@
 .quarter_width {
     width: 25%;
 }
+
+.toolFormBody {
```

Comment:
Do not worry. I removed the styles in https://github.com/galaxyproject/galaxy/pull/661
Thanks for the contributions!